hookman/hookman_utils.py

```diff
@@ -21,12 +22,26 @@ def find_config_files(plugin_dirs: Union[List[Path], Path]) -> List[Path]:
     return config_files
 
 
+@contextmanager
+def change_path_env(shared_lib_path: str):
+    """
+    Change PATH environment adding the shared library path to it.
+    """
+    old_path = os.environ["PATH"]
```

Comment:
Context managers in general should handle exceptions gracefully; also you can bail out in platforms other than Windows:

```python
    if sys.platform.startswith('win'):
        old_path = os.environ["PATH"]        
        os.environ["PATH"] = old_path + os.pathsep + os.path.dirname(shared_lib_path)
        try:
            yield
        finally:
            os.environ["PATH"] = old_path
```

Reply:
Good Ideia! 
Done.